recipes/repeatmodeler/patch

```diff
@@ -68,3 +68,107 @@ index 01fed4d..3dad631 100755
  ##---------------------------------------------------------------------------##
  ##  File:
  ##      @(#) TRFMask
+--- a/README.md
```

Comment:
What is the reason to patch the readme?

Reply:
To add information regarding NCBI's stats report and ABBlast deprecation after RepeatModeler 2.0.4 was released. I've realized this is not patch-worthy and omitted these changes in the next commit.